src/arrayMethodJoin.js

```diff
@@ -4,8 +4,20 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let str = '';
+
+    for (let i = 0; i < this.length; i++) {
+      if (this[i] !== null && this[i] !== undefined) {
+        str += this[i];
+      }
+
+      if (i !== this.length - 1) {
+        str += separator;
+      }
```

Comment:
does it work when we launch this function with argument null? I didn't pass the test. it's strange. ok, maybe I did something wrong

Reply:
![image](https://user-images.githubusercontent.com/36279782/190453940-12a8bf8d-d772-4e19-ad4f-beffb756d986.png)

Reply:
now agree. i understood when there was mistake in my code. thanks for comments ;)